src/java/htsjdk/samtools/fastq/FastqRecord.java

```diff
@@ -24,18 +24,23 @@
 package htsjdk.samtools.fastq;
 
 import java.io.Serializable;
+import java.util.Objects;
 
 /**
  * Represents a fastq record, fairly literally, i.e. without any conversion.
  */
 public class FastqRecord implements Serializable {
     private static final long serialVersionUID = 1L;
     private final String seqHeaderPrefix;
-    private final String seqLine;
+    private final String seqLine;            //not null, not empty
```

Comment:
I'm not sure...  Perhaps we should have an open ticket per release, or use the release prep features to keep a running list of things like this for the next release?  @jacarey what do you think?